networking.md

```diff
@@ -168,16 +176,13 @@ class _MyHomePageState extends State<MyHomePage> {
 
 For full API docs, see:
 
-  * [package `http`][http]
-  * [`Client` in package `http`][client]
-  * [library `dart:convert`][convert]
   * [library `dart:io`][dartio]
+  * [library `dart:convert`][convert]
 
-[http]:       https://pub.dartlang.org/packages/http
-[client]:     https://www.dartdocs.org/documentation/http/latest/http/Client-class.html
-[get]:        https://www.dartdocs.org/documentation/http/latest/http/get.html
-[read]:       https://www.dartdocs.org/documentation/http/latest/http/read.html
-[readbytes]:  https://www.dartdocs.org/documentation/http/latest/http/readBytes.html
-[post]:       https://www.dartdocs.org/documentation/http/latest/http/post.html
-[convert]:    https://docs.flutter.io/flutter/dart-convert/dart-convert-library.html
 [dartio]:     https://api.dartlang.org/stable/dart-io/dart-io-library.html
```

Comment:
For all API docs, let's link to docs.flutter.io/something

Reply:
Oh, yes of course!